spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -47,19 +47,163 @@ definitions:
      - discontinuity_counter:
             type: u8
             desc: |
-              Signal phase discontinuity counter. 
+              Signal phase discontinuity counter.
               Increased for every discontinuity in phase.
      - bias:
             type: s32
             units: 0.1 mm
             desc: Phase bias for specified signal
 
+ - SvId:
+    short_desc: Space vehicle identifier
+    desc: |
+      A (Constellation ID, satellite ID) tuple that unique identifies
+      a space vehicle
+    fields:
+    - satId:
+        type: u8
+        desc: ID of the space vehicle within its constellation
+    - constellation:
+        type: u8
+        desc: Constellation ID to which the SV belongs
+
+ - STECHeader:
+    short_desc: Header for MSG_SSR_STEC_CORRECTION message
+    desc: |
+      A full set of STEC information will likely span multiple SBP
+      messages, since SBP message a limited to 255 bytes.  The header
+      is used to tie multiple SBP messages into a sequence.
+    fields:
+    - time:
+        type: GPSTime
+        desc: GNSS time of the STEC data
+    - num_msgs:
+        type: u8
+        desc: Number of messages in the dataset
+    - seq_num:
+        type: u8
+        desc: Postion of this message in the dataset
+    - ssr_update_interval:
+        type: u16
+        desc: update interval in seconds
+    - iod_ssr:
+        type: u8
+        desc: range 0 - 15
+
+ - GriddedCorrectionHeader:
+    short_desc: Header for MSG_SSR_GRIDDED_CORRECTION
+    desc: |
+      The 3GPP message contains nested variable length arrays
+      which are not suppported in SBP, so each grid point will
+      be identified by the index.
+    fields:
+    - time:
+        type: GPSTime
+        desc: GNSS time of the STEC data
+    - num_msgs:
+        type: u8
+        desc: Number of messages in the dataset
+    - seq_num:
+        type: u8
+        desc: Postion of this message in the dataset
+    - ssr_update_interval:
+        type: u16
+        desc: update interval in seconds
+    - iod_ssr:
+        type: u8
+        desc: range 0 - 15
+    - tropo_quality:
+        type: u8
+        desc: troposphere quality indicator
+    - index:
+        type: u8
+        desc: index of the grid point
+
+ - STECSatElement:
+     short_desc: SSR code biases corrections for a particular satellite.
+     desc: |
+       Code biases are to be added to pseudorange.
```

Comment:
I think this might be the wrong description/short_desc?

Reply:
f